README.md

```diff
@@ -1,2 +1,29 @@
 # overseas-entities-api
-API for the Register An Overseas Entity service
+
+### Overview
+API service for submitting overseas entity registrations to CHIPS
+
+### Requirements
+In order to run the service locally you will need the following:
+- [Java 11](https://www.oracle.com/java/technologies/downloads/#java11)
+- [Maven](https://maven.apache.org/download.cgi)
+- [Git](https://git-scm.com/downloads)
+
+### Getting started
+To checkout and build the service:
+1. Clone [Docker CHS Development](https://github.com/companieshouse/docker-chs-development) and follow the steps in the README.
+2. Run ./bin/chs-dev modules enable overseas-entities
+3. Run ./bin/chs-dev development enable overseas-entities-api (this will allow you to make changes).
+4. Run docker using "tilt up" in the docker-chs-development directory.
+5. Use spacebar in the command line to open tilt window - wait for overseas-entities-api to become green.
+6. Open your browser and go to page http://chs.local/register-an-overseas-entity
```

Comment:
Morning, we can change it later but the endpoint right now is `/overseas-entities`, so it should be `http://chs.local/overseas-entities`

Reply:
http://chs.local/register-an-overseas-entity goes to the landing page. http://chs.local/overseas-entities gets a 404,

Reply:
http://chs.local/register-an-overseas-entity goes to the web http://chs.local/overseas-entities should go to the api. We do not have a get operation on it, so you will have 404

Reply:
These are just instruction on how we run the app in the browser so let's not get stuck on this.

Reply:
Added an extra line to cover this.